commitlint.config.ts

```diff
@@ -206,11 +206,13 @@ module.exports = {
                         if (bodyStr !== ''){
 
                             function paragraphHasValidEnding(paragraph: string): boolean {
+                                let lastWordInParagraph = paragraph.split(' ')[paragraph.split(' ').length - 1]
                                 let endingChar = paragraph[paragraph.length - 1];
                                 if (endingChar === '.' ||
                                     endingChar === ':' ||
                                     endingChar === '!' ||
-                                    endingChar === '?') {
+                                    endingChar === '?' ||
+                                    isValidUrl(lastWordInParagraph)) {
```

Comment:
it is better and more readable to call isValidUrl() before the `if` that checks endingChar

Reply:
@realmarv

Reply:
@realmarv sorry this is not done, you're still calling isValidUrl() after checking endingChar; you should be able to call it before (even before you assign endingChar at all)

Reply:
You mean like this?

Reply:
yes, that looks way more readable and maintainable, thanks

Reply:
Great!